solutions/gke/configconnector/gke-defaults/gateway-setup/dns/setters.yaml

```diff
@@ -29,8 +29,7 @@ data:
   # Client
   ##########################
   #
-  # Project that has the public dns zone.
-  # Usually the client host project that was created by the client-landing-zone package
+  # Project that has the client's public dns zone, created by the client-landing-zone package.
   # customization: required
   dns-project-id: dns-project-12345
```

Comment:
Would it be beneficial to change the setter name to `core-dns-project-id` if referencing the core dns project?

Reply:
Or is this the client host project?

Reply:
Never mind, I see that the client will get a dedicated dns project.